gensim/corpora/wikicorpus.py

```diff
@@ -255,6 +264,14 @@ def init_to_ignore_interrupt():
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
 
+def _process_article(args):
+    """Should not be called explicitly. use process_article instead"""
+    tokenizer_func, token_min_len, token_max_len, lower = args[-1]
+    args = args[:-1]
+    return process_article(args, tokenizer_func=tokenizer_func, token_min_len=token_min_len,
+                           token_max_len=token_max_len, lower=lower)
```

Comment:
Question about hanging indentation:

```
def _process_article(args):
    """Should not be called explicitly. use process_article instead."""
    
    tokenizer_func, token_min_len, token_max_len, lower = args[-1]
    args = args[:-1]
    
    return process_article(
        args, tokenizer_func=tokenizer_func, token_min_len=token_min_len, token_max_len=token_max_len, lower=lower
    )
```

looks good? If so, then there are other functions(__init__) for WikiCorpus which should be fixed as well(I can do it), if not then please ignore my comment and can you point to what the expectation is?